test/no-content.html

```diff
@@ -29,12 +29,15 @@
     <test-fixture id="juicy-html-200-empty">
         <template>
             <!-- nest to workaround test-fixture bug -->
-            <div><juicy-html href="./mock/200"></juicy-html></div>
+            <div>
+                <juicy-html href="./mock/200"></juicy-html>
+            </div>
```

Comment:
I'd keep it as it was, not to bother about text `.childNodes` when checking the stamped content.